packages/app-builder-lib/src/electron/ElectronFramework.ts

```diff
@@ -49,6 +49,12 @@ export interface ElectronDownloadOptions {
    */
   mirror?: string | null
 
+  /**
+   * The mirror base URL for Electron headers.
+   * @default https://electronjs.org/headers
+   */
+  headersMirror?: string | 'https://electronjs.org/headers'
```

Comment:
This feels like a good place to put it, but this object is passed directly as a JSON string to upstream dependency `app-builder-bin`
https://github.com/electron-userland/electron-builder/blob/443ee8debd4e4d73d9f63a31fb93ce1dbed2538c/packages/app-builder-lib/src/electron/ElectronFramework.ts#L154-L155
https://github.com/electron-userland/electron-builder/blob/443ee8debd4e4d73d9f63a31fb93ce1dbed2538c/packages/app-builder-lib/src/electron/ElectronFramework.ts#L215

The `ElectronDownloadOptions` are supposed to be specific to https://github.com/electron/get, so I'll need to think more on this feature implementation on how we can make it easy to understand for developers without convoluting the intent of each configuration property